File: day/Selection.cpp
#include<iostream>
using namespace std;

void Selection(int arr[],int size){
    
    for(int i=0;i<size-1;i++){
        int index =i;
        for(int j=j+1;j<size;j++){
            if(arr[i]>arr[j]){
                index=j;
            }
        }
        if(index!=)
    }
}
 
int main(){
    
   int arr[] = {1, 2, 3, 4, 5};
    int size = sizeof(arr) / sizeof(arr[0]);

    Selection(arr, size);

    for (int i = 0; i < size; i++)
    {
        cout << arr[i] << "\t";
    }
    return 0;
}

File: JAVASCRIPT/SORT_ALGO/Selection.js


function Selection(){
    const arr= [3, 4, 20, 10, 6];
    const size = arr.length;

    for(let i=0;i<size-1;i++){
        let index=1;

        for(let j=i+1;j<size;j++){
            if(arr[j]<arr[index]){
                index=j;
            }
        }
        let temp=arr[i];
        arr[i]=arr[index];
        arr[index]=temp;
    }
    for(let i=0;i<size;i++){
        console.log(arr[i]);
    }
}
Selection();